chromadb/errors.py

```diff
@@ -81,6 +81,13 @@ def name(cls) -> str:
         return "InvalidHTTPVersion"
 
 
+class OutOfBoundError(ChromaError):
```

Comment:
Perhaps "InvalidArgument" is more generically useful and also give user clearer information.

Also, can we please add a code - 400 in this case?

Reply:
I've renamed the error to `InvalidArgumentError`. The inherited error code from `ChromaError` should be `400` already, but I've added the code for explicitness now.

Reply:
Great, thanks